### PR TITLE
Pull Request: Set-up Enzyme Test Suite

### PR DESCRIPTION
**Pull request by martinsung00**
Added dependencies for Enzyme. Initial set-up and configurations. Separated server/DB tests from client tests. Added scripts to run tests. Added dummy describe-test. Remove dead code in seeder helper.

### CODE REVIEW

File: test/backend_spec/seederUnitTest.js [hunk 1]
```diff
@@ -76,12 +51,10 @@ describe('Helper Functions', function () {
...
   describe('Random Rating', function () {
-    const ratings = [0, 1, 2, 3, 4, 5];
-    it('should generate a random number between 1 to 5', function () {
+    it('should generate a random number', function () {
       const randomRating = helper.randomRating();
-      const index = ratings.indexOf(randomRating);
-      const truthiness = index > -1;
-      assert.equal(true, truthiness);
+
+      assert.equal('number', typeof randomRating);
```
Comment (anabellchan) [inline test/backend_spec/seederUnitTest.js:57]: Nice work on testing your random function.  You may also test that the number is in a specific range if you want to increase test precision.

File: test/client_spec/reviews.spec.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+/* eslint-disable import/named */
+/* eslint-disable prefer-arrow-callback */
+import React from 'react';
+import Reviews from '../../react-client/src/components/Reviews';
+import styles from '../../react-client/src/styles/Reviews.module.css';
+
+describe('Reviews', function () {
+  it('Should render the Reviews wrapper', function () {
+    const wrapper = shallow(<Reviews />);
+    expect(wrapper.hasClass(`.${styles.parentDiv}`)).to.equal(true);
```
Comment (anabellchan) [inline test/client_spec/reviews.spec.js:14]: Well done!

**Review by anabellchan — APPROVED** [review]
Good to go!

File: test/backend_spec/seederUnitTest.js (hunk 1 above)
Comment (martinsung00) [reply test/backend_spec/seederUnitTest.js:57]: Thanks for the suggestion! I originally had a test to check if the number is in the specific range but I thought it was a little extra so I removed it.

**Pull request merged**